### PR TITLE
[CIAPP-6905] Adding CI Visibility audit events.

### PR DESCRIPTION
**Pull request by rodrimendoza**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by drodriguezhdez — APPROVED** [review]

**Review by TovRudyy — APPROVED** [review]

File: content/en/account_management/audit_trail/events.md [hunk 1]
```diff
@@ -101,9 +101,13 @@ See the [Audit Trail documentation][2] for more information on setting up and co
...
-|---------------------------------|--------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
-| [Repository Default Branch][30] | A user modified the default branch of a repository, and the previous and new values for the default branch. | `@evt.name:"CI Visibility" @asset.type:ci_app_repository`   
+| Name                            | Description of audit event                                                                                  | Query in audit explorer                                              |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|
+| [Repository Default Branch][30] | A user modified the default branch of a repository, and the previous and new values for the default branch. | `@evt.name:"CI Visibility" @asset.type:ci_app_repository`            |
```
Comment (rtrieu) [inline content/en/account_management/audit_trail/events.md:106]:
```suggestion
| [Repository Default Branch][30] | A user modified the default branch of a repository, as well as the previous and new values for the default branch. | `@evt.name:"CI Visibility" @asset.type:ci_app_repository`            |
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Added a minor suggestion, but otherwise looks good!

File: content/en/account_management/audit_trail/events.md (hunk 1 above)
Comment (rodrimendoza) [reply content/en/account_management/audit_trail/events.md:106]: Hi Rosa, I'm actually not changing this description in this PR. But it seems strange, given that the user is not modifying 'the previous and new values for the default branch', it's just updating the default branch. In any case, the previous and the new values are shown as part of the audit event. I may rephrase it as:

> A user modified the default branch of a repository.

What do you think?

Comment (rtrieu) [reply content/en/account_management/audit_trail/events.md:106]: That works and seems more clear/succinct! :)

Comment (rodrimendoza) [reply content/en/account_management/audit_trail/events.md]: Great, I've pushed it :)

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**